### PR TITLE
mutation->extension

### PR DESCRIPTION
**Pull request by rainmaker022**
Changes user facing text from mutation(s) to extension(s)

### CODE REVIEW

**Comment by rainmaker022** [comment]
Im not sure why there are so many deletions

**Comment by Flaque** [comment]
The deletions are just Yarn updating the lock file. It's fine <3 

Also it looks like you just need to update the snapshots

File: src/lib/paths/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export function auroraPreferencesFile() {
 
 // ~/.aurora/<NODE_ENV>/mutations
 export function auroraMutationPath() {
-  return path.join(auroraUserPath(), "mutations");
+  return path.join(auroraUserPath(), "extensions");
```
Comment (kmccrohan) [inline src/lib/paths/index.js]: Not sure we want to change this path. This is where we store mutations. Don't change this.

Comment (Flaque) [reply src/lib/paths/index.js]: I mean it's fine if it doesn't break anything

**Comment by kmccrohan** [comment]
LGTM!

**Pull request merged**